Makefile

```diff
@@ -1,4 +1,4 @@
-DOCKERHUB_REPO := taccwma/$(shell cat ./docker_repo.var)
```

Comment:
We need to keep this line using the configurable `./docker_repo.var` setting.

Why:
The value in that file is used when generating the dockerhub image name that is unique to each portal.

Details:
While all portals share a common `core-portal` container, the CMS container is unique per portal due to the embedded branding and custom navigation elements, templates, styles, etc. 

The string value in the`docker_repo.var` file is used in the CMS codebase to:
- correlate the contents of the branding submodule in the `Core-CMS-Resources` repo 
- determine the name of the published  dockerhub image and target repo

We generate the CMS dockerhub images using the nomenclature of `$PROJECT-cms`. If you look under the taccwma dockerhub repos (https://hub.docker.com/orgs/taccwma/repositories), you will see `a2cps-cms`, `protx-cms`, `neuronex-cms`, etc. which comes from the value placed in the local docker_repo.var` file during the image build phase of the Jenkins job (here under the `Build` section: http://jenkins01.tacc.utexas.edu:8080/job/Core_CMS/configure)

Reply:
Good catch! I'll update the Makefile to restore the functionality